src/repositories_rust.tf

```diff
@@ -116,6 +116,19 @@ locals {
       },
       "telemetry" = {
         description = "Receive and rebroadcast vehicle and vertiport telemetry."
+        # Override files list to provision with Terraform
+        # svc-telemetry handles its own docker-compose.yml file
+        # due to extra backend services needed
+        files = merge(
```

Comment:
Just like for svc-storage, you'll have to provide all non-default rust files (all workflows listed for the storage service), or they won't be provisioned. They're part of the svc specific files list, which you can't merge in unless we define a separate local var for that list as well.

Reply:
https://github.com/Arrow-air/tf-github/commit/748ce1fd1c3a5c8abaef27a2223f1f20ae46c153 fixed